modules/040-control-plane-manager/images/control-plane-manager/werf.inc.yaml

```diff
@@ -21,15 +21,23 @@ docker:
   ENTRYPOINT: ["/controller"]
 {{- end }}
 ---
-artifact: {{ $.ModuleName }}/controller-artifact
-from: {{ $.Images.BASE_GOLANG_22_ALPINE_DEV }}
-fromCacheVersion: 1
+artifact: {{ $.ModuleName }}-{{ $.ImageName }}-src-artifact
```

Comment:
we use another naming -  {{ $.ModuleName }}/{{ $.ImageName }}-src-artifact

Reply:
fixed